libs/ui-lib/lib/cim/components/modals/ProvisioningConfigErrorAlert.tsx

```diff
@@ -0,0 +1,17 @@
+import { Alert } from '@patternfly/react-core';
+import React from 'react';
+import { useTranslation } from '../../../common/hooks/use-translation-wrapper';
+
+const ProvisioningConfigErrorAlert = ({ error }: { error: unknown }) => {
+  const { t } = useTranslation();
+  if (!error) {
+    return null;
+  }
+  return (
+    <Alert title={t('ai:Failed to get Provisioning Configuration')} variant="danger" isInline>
+      {(error as Error).message || ''}
```

Comment:
to get the err msg, you could use `getErrorMessage` from `common/utils`

Reply:
fixed